internal/utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"strconv"
+	"strings"
+)
+
+// IntArrayToString converts an array of integers
+// to a comma-separated list string -
+// e.g [1, 2, 3] will be converted to "1,2,3".
+func IntArrayToString(integers []int) string {
+	sArray := []string{}
+
+	for _, n := range integers {
+		sArray = append(sArray, strconv.Itoa(n))
+	}
+
+	return strings.Join(sArray, ",")
+}
```

Comment:
Looping through append could be optimized since we know the length of `integers`:

```
func IntArrayToString(integers []int) string {
	sArray := make([]string, len(integers))

	for i, n := range integers {
		sArray[i] = strconv.Itoa(n)
	}

	return strings.Join(sArray, ",")
}
```

Reply:
Updated, thanks!